18/README.md

```diff
@@ -0,0 +1,128 @@
+```
+shortname: 18/tm_stress
```

Comment:
BEP-18 is already taken by a previous PR https://github.com/bigchaindb/BEPs/pull/44

Reply:
I think BEP-20 is available as BEP-19 is taken too.